src/modules/API/counter.js

```diff
@@ -0,0 +1,8 @@
+const counter = document.getElementById('counter');
+
```

Comment:
- [ ] As required for the project, kindly ensure that you to still create a specific function to calculate `counter numbers` on each page even if the API provides that information. As right now, I noticed that you miss the `LikesCounter` function for the Home page. Make sure you add it

Reply:
@tresorsawasawa ,

Kindly help us understand what you meant when you suggested when implementing a likes function counter yet we already have it.

Reply:
Hi @zeff96 

For that should the same way you did for `ReservationsCounter` and `CommentsCounter`. Just Refer the `src/modules/comments/commentCounter.js` and `src/modules/reservation/reserveCounter.js` files as well and do the same for `LikesCounter`